frontend/device/kindle/powerd.lua

```diff
@@ -29,6 +29,7 @@ function KindlePowerD:toggleFrontlight()
     if sysint == 0 then
         self:setIntensity(self.fl_intensity)
```

Comment:
I think a better fix would be keeping the check in setIntensity and replace this call with self:setIntensityHW() here ;)